src/vue/common/chart/Chart.vue

```diff
@@ -74,14 +76,25 @@ export default {
   },
   watch: {
     async lockedAssets (value) {
+      this.createIntervalForLoadPrices()
       await this.loadPrices()
     },
   },
+  async beforeDestroy () {
+    clearInterval(this.intervalID)
```

Comment:
since `intervalId` is only used inside the `createIntervalForLoadPrices()`, clearing the interval id in this way breaks encapsulation. should be extracted in a method like "clearLoadPricesTicker()"